libs/ledger-live-common/src/families/bitcoin/wallet-btc/explorer/index.ts

```diff
@@ -163,55 +160,42 @@ class BitcoinLikeExplorer implements IExplorer {
     return parseFloat(fees["relay_fee"]);
   }
 
-  async getPendings(address: Address, nbMax?: number): Promise<TX[]> {
-    const params: ExplorerParams =
-      this.explorerVersion === "v2"
-        ? {
-            noToken: "true",
-          }
-        : {
-            no_token: "true",
-          };
-    if (!this.disableBatchSize) {
-      params.batch_size = nbMax || 1000;
-    }
-    const res = await this.fetchTxs(address, params);
-    const pendingsTxs = res.txs.filter((tx) => !tx.block);
+  async getPendings(address: Address, nbMax?: number) {
+    const params: ExplorerParams = {
+      no_token: "true",
+      batch_size: !this.disableBatchSize ? nbMax || 1000 : undefined,
+      block_hash: undefined,
+    };
+    const txs = await this.fetchTxs(address, params);
+    const pendingsTxs = txs.filter((tx) => !tx.block);
     pendingsTxs.forEach((tx) => this.hydrateTx(address, tx));
     return pendingsTxs;
   }
 
-  async fetchTxs(
+  async fetchTxs(address: Address, params: ExplorerParams): Promise<TX[]> {
+    const url = `/address/${address.address}/txs`;
```

Comment:
very minor detail but if we are exact this is more an endpoint and not an url

Reply:
This term has already been used for the same reason across the codebase, i don't feel like doing an exception